libwallet/internal/politeia/politeia_sync.go

```diff
@@ -334,6 +335,7 @@ func (p *Politeia) fetchBatchProposals(category int32, tokens []string, broadcas
 				return p.ctx.Err()
 			}
 
+			proposals[i].LowerName = strings.ToLower(proposals[i].Name)
```

Comment:
Er. No. It's usually a sign of bad database design if you have to store the same value in 2 fields. You can use the `Name` field for search queries. See my next comment.